micro-rdk-nmea/build.rs

```diff
@@ -0,0 +1,154 @@
+use serde_json::Value;
+use std::fs::{create_dir_all, File, OpenOptions};
+use std::io::{BufReader, Write};
+use std::path::Path;
+
+use convert_case::{Case, Casing};
+
+// This is to compute a valid name for a variable or struct/enum member. We
+// want to remove special characters and introduce a default convention for
+// when a name starts with a number.
+fn clean_string_for_rust(input: &str, counter: usize) -> String {
+    let filtered: String = input
+        .chars()
+        .map(|c| {
+            if c.is_alphanumeric() || c.is_whitespace() {
+                c
+            } else {
+                '_'
```

Comment:
what non alphanumeric character we running into? Should we just filter them out instead of replacing?

Reply:
I found that the replacement with underscore made swapping to pascal/snake case easier in the case of a name like "@23..."